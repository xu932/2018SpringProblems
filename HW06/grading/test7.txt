id:1 name:Natividad major:CHE year:Freshman enrollment:No age:21 
id:2 name:Eyian major:CHE year:Senior enrollment:Yes age:20 
id:3 name:Cosme major:AA year:Junior enrollment:No age:22 
id:4 name:Rhyla major:BME year:Sophomore enrollment:Yes age:19 
id:5 name:Premier major:ECE year:Junior enrollment:No age:22 
id:6 name:Camp major:IE year:Senior enrollment:No age:21 
id:7 name:Redding major:BME year:Senior enrollment:No age:20 
id:8 name:Rennick major:CS year:Freshman enrollment:No age:21 
id:9 name:Ziyang major:ME year:Freshman enrollment:No age:19 
