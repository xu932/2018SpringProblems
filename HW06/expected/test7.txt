id:1 name:Natividad major:CHE year:Freshman enrollment:No age:21 
id:3 name:Cosme major:AA year:Junior enrollment:No age:22 
id:5 name:Premier major:ECE year:Junior enrollment:No age:22 
id:6 name:Camp major:IE year:Senior enrollment:No age:21 
id:7 name:Redding major:BME year:Senior enrollment:No age:20 
id:8 name:Rennick major:CS year:Freshman enrollment:No age:21 
id:9 name:Ziyang major:ME year:Freshman enrollment:No age:19 
id:13 name:Jerzie major:CHE year:Freshman enrollment:No age:19 
id:17 name:Javan major:AA year:Freshman enrollment:No age:21 
id:18 name:Evina major:ABE year:Junior enrollment:No age:19 
id:20 name:Nili major:ECE year:Sophomore enrollment:No age:19 
id:21 name:Aleksandr major:CS year:Senior enrollment:No age:20 
id:22 name:Rohan major:IE year:Junior enrollment:No age:19 
id:23 name:Lynox major:CS year:Sophomore enrollment:No age:19 
id:25 name:Aedin major:CS year:Sophomore enrollment:No age:20 
id:26 name:Sayuri major:CHE year:Junior enrollment:No age:22 
id:28 name:Bradly major:ECE year:Junior enrollment:No age:20 
id:29 name:Meridian major:CHE year:Junior enrollment:No age:21 
id:30 name:Cherilyn major:CS year:Freshman enrollment:No age:19 
id:33 name:Mehjabeen major:IE year:Senior enrollment:No age:18 
id:35 name:Keliah major:ECE year:Senior enrollment:No age:20 
id:36 name:Malhar major:IE year:Senior enrollment:No age:21 
id:39 name:Jocee major:IE year:Junior enrollment:No age:21 
id:47 name:Samatar major:IE year:Freshman enrollment:No age:21 
id:48 name:Collyn major:ECE year:Sophomore enrollment:No age:20 
id:49 name:Evalisse major:CHE year:Freshman enrollment:No age:21 
id:52 name:Keilynn major:CS year:Freshman enrollment:No age:20 
id:54 name:Analeigh major:AA year:Senior enrollment:No age:20 
id:56 name:Ramy major:CHE year:Freshman enrollment:No age:19 
id:58 name:Amiyah major:CS year:Sophomore enrollment:No age:19 
id:59 name:Jadah major:IE year:Sophomore enrollment:No age:21 
id:60 name:Maxwel major:CHE year:Senior enrollment:No age:22 
id:61 name:Virgil major:ABE year:Senior enrollment:No age:22 
id:65 name:Ruger major:AA year:Junior enrollment:No age:18 
id:67 name:Airess major:BME year:Sophomore enrollment:No age:22 
id:68 name:Kyce major:AA year:Freshman enrollment:No age:18 
id:69 name:Araia major:CHE year:Senior enrollment:No age:20 
id:71 name:Kol major:CHE year:Senior enrollment:No age:21 
id:72 name:Joelie major:BME year:Junior enrollment:No age:22 
id:75 name:Viera major:ABE year:Senior enrollment:No age:22 
id:78 name:Braelyn major:BME year:Senior enrollment:No age:21 
id:80 name:Lanah major:ABE year:Sophomore enrollment:No age:22 
id:81 name:Riah major:CHE year:Freshman enrollment:No age:18 
id:84 name:Colten major:ECE year:Sophomore enrollment:No age:22 
id:85 name:Shani major:ECE year:Freshman enrollment:No age:21 
id:87 name:Yafa major:CS year:Senior enrollment:No age:22 
id:98 name:Breighlyn major:CS year:Junior enrollment:No age:21 
id:99 name:Sadieann major:CS year:Freshman enrollment:No age:22 
id:103 name:Aliani major:CHE year:Senior enrollment:No age:20 
id:107 name:Maziya major:CHE year:Senior enrollment:No age:21 
id:109 name:Jamarious major:ECE year:Freshman enrollment:No age:22 
id:111 name:Yohandri major:AA year:Sophomore enrollment:No age:22 
id:112 name:Alaysha major:ECE year:Sophomore enrollment:No age:20 
id:113 name:Azayla major:ME year:Sophomore enrollment:No age:18 
id:116 name:Melquisedec major:ME year:Junior enrollment:No age:21 
id:122 name:Natalia major:CS year:Freshman enrollment:No age:18 
id:125 name:Truett major:ABE year:Freshman enrollment:No age:20 
id:130 name:Ramaya major:ECE year:Senior enrollment:No age:20 
id:132 name:Salah major:IE year:Junior enrollment:No age:19 
id:133 name:Arilynn major:ECE year:Sophomore enrollment:No age:18 
id:135 name:Stellaluna major:ABE year:Junior enrollment:No age:19 
id:136 name:Alys major:AA year:Freshman enrollment:No age:22 
id:139 name:August major:CS year:Junior enrollment:No age:22 
id:141 name:Ilithyia major:AA year:Sophomore enrollment:No age:18 
id:144 name:Jaeshaun major:ABE year:Freshman enrollment:No age:22 
id:146 name:Isileli major:AA year:Junior enrollment:No age:21 
id:147 name:Saiyan major:ABE year:Freshman enrollment:No age:22 
id:148 name:Kashlyn major:IE year:Junior enrollment:No age:18 
id:149 name:Joylynn major:CS year:Freshman enrollment:No age:20 
id:151 name:Lema major:CS year:Sophomore enrollment:No age:18 
id:154 name:Brigg major:ABE year:Sophomore enrollment:No age:21 
id:155 name:Damontay major:ECE year:Sophomore enrollment:No age:21 
id:157 name:Nereida major:AA year:Sophomore enrollment:No age:20 
id:158 name:Odinn major:IE year:Senior enrollment:No age:18 
id:159 name:Vansh major:ABE year:Junior enrollment:No age:19 
id:160 name:Aiydan major:CHE year:Senior enrollment:No age:20 
id:164 name:Lexander major:ECE year:Senior enrollment:No age:20 
id:165 name:Seleny major:ABE year:Freshman enrollment:No age:20 
id:166 name:Kaylianna major:BME year:Senior enrollment:No age:20 
id:168 name:Suhaira major:ECE year:Senior enrollment:No age:21 
id:169 name:Nikoli major:IE year:Sophomore enrollment:No age:21 
id:171 name:Kanishk major:IE year:Freshman enrollment:No age:19 
id:173 name:Sonam major:CHE year:Freshman enrollment:No age:19 
id:175 name:Micaella major:ABE year:Sophomore enrollment:No age:21 
id:176 name:Damiyah major:CS year:Sophomore enrollment:No age:19 
id:178 name:Karsyn major:ECE year:Junior enrollment:No age:22 
id:179 name:Dee major:AA year:Sophomore enrollment:No age:20 
id:180 name:Jaretzi major:BME year:Senior enrollment:No age:21 
id:181 name:Nakaylah major:CHE year:Freshman enrollment:No age:20 
id:182 name:Yisrael major:BME year:Senior enrollment:No age:22 
id:183 name:Kyler major:ECE year:Freshman enrollment:No age:19 
id:185 name:Azaylia major:ECE year:Sophomore enrollment:No age:22 
id:187 name:Ambrosia major:BME year:Sophomore enrollment:No age:18 
id:188 name:Marine major:ECE year:Sophomore enrollment:No age:18 
id:189 name:Daeson major:BME year:Senior enrollment:No age:18 
id:190 name:Berel major:ECE year:Senior enrollment:No age:19 
id:191 name:Jovonte major:ME year:Freshman enrollment:No age:18 
id:193 name:Denilson major:BME year:Freshman enrollment:No age:18 
id:197 name:Raedyn major:BME year:Sophomore enrollment:No age:19 
id:198 name:Remick major:ECE year:Senior enrollment:No age:21 
id:200 name:Lovely major:BME year:Freshman enrollment:No age:21 
